bot/seasons/evergreen/game.py

```diff
@@ -0,0 +1,323 @@
+
+        body += f"sort {sort};\n" if sort else ''
+        body += f"offset {offset};\n"
+        body += f"limit {limit};\n"
+
+        body += f"where genres = ({genre});" if genre else ''
+        body += additional_body
+
+        # Do request to IGDB API, create headers, URL, define body, return result
+        async with client.get(
+                url=BASE_URL + "games",
+                data=body,
+                headers=HEADERS
+        ) as resp:
+            return await resp.json()
+
+    async def create_page(self, data: Dict[str, Any]) -> Tuple[str, str]:
+        """Create content of Game Page."""
+        # Create page content variable, what will be returned
+        page = ""
+
+        # If game have cover, generate URL of Cover, if not, let url empty
+        if 'cover' in data:
+            url = f"{IMAGE_BASE_URL}t_cover_big/{data['cover']['image_id']}.jpg"
+        else:
+            url = ""
+
+        # Add title with hyperlink and check for storyline
+        page += f"**[{data['name']}]({data['url']})**\n"
+        page += data['summary'] + "\n\n" if 'summary' in data else "\n"
+
+        # Add release date if key is in game information
+        if 'first_release_date' in data:
+            page += f"**Release Date:** {datetime.utcfromtimestamp(data['first_release_date']).date()}\n"
+
+        # Add other information
+        page += f"**Rating:** {'{0:.2f}'.format(data['total_rating']) if 'total_rating' in data else '?'}/100 "
+        page += f":star: (based on {data['total_rating_count'] if 'total_rating_count' in data else '?'})\n"
+
+        page += f"**Platforms:** "
+        page += f"{', '.join(pf['name'] for pf in data['platforms']) if 'platforms' in data else '?'}\n"
+
+        page += f"**Status:** {GameStatus(data['status']).name if 'status' in data else '?'}\n"
+
+        if 'age_ratings' in data:
+            rating = ', '.join(f"{AgeRatingCategories(age['category']).name} {AgeRatings(age['rating']).name}"
+                               for age in data['age_ratings'])
+            page += f"**Age Ratings:** {rating}\n"
+
+        if 'involved_companies' in data:
+            companies = ', '.join(co['company']['name'] for co in data['involved_companies'])
+        else:
+            companies = "?"
+        page += f"**Made by:** {companies}\n"
+
+        page += "\n"
+        page += data['storyline'] if 'storyline' in data else ''
+
+        return page, url
+
+    async def search_games(self, client: ClientSession, search: str) -> List[str]:
+        """Search game from IGDB API by string, return listing of pages."""
+        lines = []
+
+        # Define request body of IGDB API request and do request
+        body = f"""fields name, url, storyline, total_rating, total_rating_count;
+search "{search}";
+limit 50;"""
+
+        async with client.get(
+                url=BASE_URL + "games",
+                data=body,
+                headers=HEADERS) as resp:
```

Comment:
On line 297, you style a similar expression in a preferred manner:
```python
async with client.get(
    url=BASE_URL + "companies",
    data=body,
    headers=HEADERS
) as resp:
```
Without getting into a discussion of which one is more "correct", it's good to be consistent.

By the way, this whole thing will comfortably fit into one line:
```python
async with client.get(url=BASE_URL + "companies", data=body, headers=HEADERS) as resp:
```